SPIKES.md

```diff
@@ -0,0 +1,89 @@
+## Writing a Spike at `conda`
+
+### Summary
+
+In this document, we go over the procedure to follow when creating spikes. Exactly what
+these types of issues are and the steps to take when creating them are all outlined
+below. These guidelines apply for all projects falling underneath the
+[conda](github.com/conda) organization.
+
+### What is a spike?
+
+A spike is a term that is borrowed from extreme programming and agile development.
+They are used when the **outcome of a ticket is unknown or even optional**. For example,
+when first coming across a problem that has not been solved before, a team may choose
+to either research the problem or create a prototype in order to better understand it.
+
+Additionally, spikes represent work that **may or may not actually be completed
+or implemented**. An example of this are prototypes created to explore possible solutions.
+Not all prototypes are implemented and the purpose of creating a prototype
+is often to explore the problem space more. For research oriented tasks, the end result
+of this research may be that a feature request simply is not viable at the moment and
+would result in putting a stop to that work.
+
+Finally, spikes are **time boxed**. This means that we set a definite limit on how long
```

Comment:
```suggestion
Finally, spikes are **timeboxed**. This means that we set a definite limit on how long
```
"Timebox" is written as one word in the rest of this doc so I'm suggesting this change to keep things consistent

Reply:
That word doesn't appear in my dictionary, so I'm going to switch to using the two word version.